starcheck/utils.py

```diff
@@ -45,6 +46,21 @@
     message=r"\nModel .* computed between .* clipping input mag\(s\) outside that range\.",
 )
 
+def prehtml2text(html_text):
+    """Convert the starcheck report html to plain text."""
+
+    soup = BeautifulSoup(html_text, "lxml")
+
+    # All of the report is basically in the pre tags
+    pres = list(soup.find_all("pre"))
+    out = []
+    for pre in pres:
+        out.append(pre.get_text())
+        # And include a line of equals to separate the sections
+        out.append("=" * 84)
```

Comment:
Look for list comprehensions when you can (and ask copilot for help!). Also try to stick to the style recommendation of using a plural for a list. 
```suggestion
section_separator = "\n" + "=" * 84
outs = [pre.get_text() + section_separator for pre in soup.find_all("pre")]
```

Reply:
Sure!  Thanks for the change.  When I was spitting out the text, it was handier to have it in the loop for debugging and checking a section at a time, but it is reasonable to compress the code a bit when that debugging is done.